examples/avro-cli.py

```diff
@@ -79,28 +92,31 @@ def on_delivery(err, msg, obj):
             obj.id, msg.topic(), msg.partition(), msg.offset()))
 
 
-def produce(topic, conf):
+def produce_msgs(topic, conf, avro_serializer):
     """
         Produce User records
     """
 
-    from confluent_kafka.avro import AvroProducer
-
-    producer = AvroProducer(conf, default_value_schema=record_schema)
+    producer = Producer(conf,
+                        key_serializer=StringSerializer('utf_8'),
```

Comment:
Make sure to document on the Producer class that it expects a BaseSerializer **instance** and not **class**.